stackit/internal/services/argus/instance/resource.go

```diff
@@ -500,6 +593,37 @@ func mapFields(ctx context.Context, r *argus.GetInstanceResponse, model *Model)
 		model.OtlpTracesURL = types.StringPointerValue(i.OtlpTracesUrl)
 		model.ZipkinSpansURL = types.StringPointerValue(i.ZipkinSpansUrl)
 	}
+
+	err := mapACLField(acl, model)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func mapACLField(aclList *argus.ListACLResponse, model *Model) error {
+	if aclList == nil {
+		if model.ACL.IsNull() || model.ACL.IsUnknown() {
+			model.ACL = types.SetNull(types.StringType)
+		}
+		return nil
+	}
+
+	if aclList.Acl == nil || len(*aclList.Acl) == 0 {
+		model.ACL = types.SetNull(types.StringType)
+		return nil
+	}
+
+	acl := []attr.Value{}
+	for _, cidr := range *aclList.Acl {
+		acl = append(acl, types.StringValue(cidr))
+	}
+	aclsList, diags := types.SetValue(types.StringType, acl)
```

Comment:
Maybe we can find a more descriptive name for this `aclsList` variable. `aclTF`? Or `aclSet`?